x11/mesa/Portfile

```diff
@@ -110,6 +111,19 @@ if {[string match *gcc* ${configure.compiler}]} {
     configure.cxxflags-append -fpermissive
 }
 
+variant llvm {
+    set llvm_ver            19
+    depends_lib-append      port:llvm-${llvm_ver}
+    # for llvm-config
+    configure.env-append    PATH=${prefix}/libexec/llvm-${llvm_ver}/bin:$env(PATH)
+    # for llvm-ar
+    build.env-append        PATH=${prefix}/libexec/llvm-${llvm_ver}/bin:$env(PATH)
+    configure.args-replace  -Dgallium-drivers=softpipe \
+                            -Dgallium-drivers=softpipe,llvmpipe
+}
+
+default_variants        +llvm
```

Comment:
Could it be properly conditional, so that a user would not need force disable it and get weird errors trying to install some dependent of `mesa`?
I do not know which x86 systems support llvm19 (perhaps not 10.4, for example), but for sure this must not be a default on powerpc.

Reply:
Can you provide the condition?